src/Loader.php

```diff
@@ -187,6 +215,18 @@ protected function looksLikeSetter($line)
         return strpos($line, '=') !== false;
     }
 
+	/**
+     * Determine if the given line looks like it's setting a variable from a Docker secret.
+     *
+     * @param string $line
+     *
+     * @return bool
+     */
+    protected function looksLikeDockerSecret($line)
+    {
+    	return strpos($line, '||') !== false;
```

Comment:
won't this fail with a line that contains a variable with the delimiter `||` i.e. `test="testing||this"` ?
possibly change the test for a line ending in `$docker` ? this removes any false positives; any envvar ending in `$docker` is read from the the other file...

Reply:
The delimiter must be at the end, so having "testing||this" wouldn't invoke the docker secret.

Reply:
it would as it is a a check that the `line` _contains_ a `||`.  The line can be something like `test=this` . `this` can contain a `||`

i.e. `test="this||"`